react-ui/src/app/dashboard/household/components/form/head-of-household.js

```diff
@@ -30,6 +30,18 @@ const HeadOfHousehold = () => {
                   validator={requiredValidator}
                 />
               </Col>
+	      <Col md={4} xs={12}>
+                <Input
+                    label="Number of Children to Nominate"
```

Comment:
@dallen72 Could you change this label to "Number of children nominated?" (Minus the question mark)

Unless @Jillzey would prefer _another_ label?